SWLOR.Game.Server/Feature/RecipeDefinition/FabricationRecipeDefinition/FurnitureRecipes.cs

```diff
@@ -2790,6 +3643,83 @@ private void Tier5()
+                .Category(RecipeCategoryType.Fixtures)
+                .Resref("structure_0326")
+                .Level(52)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 5)
+                .RequirementPerk(PerkType.StructureBlueprints, 2)
+                .RequirementUnlocked()
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("r_pow_supp_unit", 2)
+                .Component("r_const_parts", 5)
+                .Component("em_amp_4", 2)
+                .Component("hull_boost_4", 2)
+                .Component("elec_high", 40)
+                .Component("ref_arkoxit", 20)
+                .Component("emerald", 10)
+                .Component("zinsiam", 5);
+
+            // Swoop Bike, Grey
+            _builder.Create(RecipeType.SwoopBikeGrey, SkillType.Fabrication)
+                .Category(RecipeCategoryType.Fixtures)
+                .Resref("structure_0327")
+                .Level(52)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 5)
+                .RequirementPerk(PerkType.StructureBlueprints, 2)
+                .RequirementUnlocked()
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("r_pow_supp_unit", 2)
+                .Component("r_const_parts", 5)
+                .Component("em_amp_4", 2)
+                .Component("hull_boost_4", 2)
+                .Component("elec_high", 40)
+                .Component("ref_arkoxit", 20)
+                .Component("emerald", 10)
+                .Component("zinsiam", 5);
+
+            // Swoop Bike, Red
+            _builder.Create(RecipeType.SwoopBikeRed, SkillType.Fabrication)
+                .Category(RecipeCategoryType.Fixtures)
+                .Resref("structure_0328")
+                .Level(52)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 5)
+                .RequirementPerk(PerkType.StructureBlueprints, 2)
+                .RequirementUnlocked()
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("r_pow_supp_unit", 2)
+                .Component("r_const_parts", 5)
+                .Component("em_amp_4", 2)
+                .Component("hull_boost_4", 2)
+                .Component("elec_high", 40)
+                .Component("ref_arkoxit", 20)
+                .Component("emerald", 10)
+                .Component("zinsiam", 5);
+
+            // Swoop Bike, Yellow
+            _builder.Create(RecipeType.SwoopBikeYellow, SkillType.Fabrication)
+                .Category(RecipeCategoryType.Fixtures)
+                .Resref("structure_0329")
+                .Level(52)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 5)
+                .RequirementPerk(PerkType.StructureBlueprints, 2)
+                .RequirementUnlocked()
```

Comment:
There looks to be 41 different RequirementUnlocked added in this file. However, I'm not seeing any update to loot tables to add the recipes to drops anywhere. How are they unlocked?

Reply:
All unlocked via quests that reward recipes.